packages/ui/src/components/Channel/index.tsx

```diff
@@ -119,7 +117,7 @@ export function Channel({
                       >
                         <Spinner />
                       </View>
-                    ) : (
+                    ) : channel.type === 'chat' ? (
```

Comment:
We have to include `dm` and `groupDm` channel types as well. Same for showing the `MessageInput`